en/views/helpers/form.rst

```diff
@@ -1767,7 +1767,7 @@ control the generated legend/fieldset.
 
 For example::
 
-    echo $this->Form->allInputs(
+    echo $this->Form->allControls(
```

Comment:
Keeping both versions with  version note is better IMO and in line with most of what has been done in similar cases